packages/backend/src/open_payments/payment_pointer/service.ts

```diff
@@ -102,6 +119,29 @@ async function createPaymentPointer(
   }
 }
 
+async function updatePaymentPointer(
+  deps: ServiceDependencies,
+  { id, status, publicName }: UpdateOptions
+): Promise<PaymentPointer | PaymentPointerError> {
+  try {
+    const update: UpdateInput = { publicName }
+    if (status) {
+      update.deactivatedAt =
+        status === PaymentPointerStatus.Inactive ? new Date() : null
```

Comment:
```suggestion
        status === 'INACTIVE' ? new Date() : null
```
so we can remove `PaymentPointerStatus` enum from here

Reply:
good catch, should have done this with the previous suggestion on this topic. removed along with import and usage in the service tests here: https://github.com/interledger/rafiki/pull/1417/commits/c973590975e09c4c0a6088456779f5f79151bfc5